src/components/NFTCover.vue

```diff
@@ -31,11 +31,17 @@
         @error="handleImageError"
       />
       <img
-        v-if="!isShowVideo && (!isShowImage || !isLoaded)"
+        v-if="!isShowVideo && (!isShowImage || !isLoaded) && !isCollection"
         v-bind="imgProps"
         :class="imgClassNameForPlaceholder"
         src="~/assets/images/nft/primitive-nft.jpg"
       />
+      <img
+        v-if="!isShowVideo && (!isShowImage || !isLoaded) && isCollection"
+        v-bind="imgProps"
+        :class="imgClassNameForPlaceholder"
+        src="~/assets/images/nft/collection.png"
+      />
```

Comment:
Can we do `isCollection ? 'collection.png' : 'nft.jpg'` with `:src=`?

Reply:
[💄 Add collection default cover](https://github.com/likecoin/liker-land/pull/1539/commits/db609a144e27cdabb4c00b1df9040102746bad0f)